lib/components/regions_map/components/serach_places/search_places.dart

```diff
@@ -3,13 +3,15 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:seeds/components/regions_map/components/serach_places/view_models/search_places_bloc.dart';
 import 'package:seeds/components/regions_map/interactor/view_models/place.dart';
 import 'package:seeds/components/regions_map/interactor/view_models/regions_map_bloc.dart';
+import 'package:seeds/datasource/remote/model/region_model.dart';
 import 'package:seeds/design/app_colors.dart';
 import 'package:seeds/design/app_theme.dart';
 
 class SearchPlaces extends StatefulWidget {
+  final List<RegionModel> regions;
```

Comment:
is there a reason why these two 
final List<RegionModel> regions;
final ValueSetter<Place> onPlaceSelected;

Are not in the state and we make this widget stateless?

Reply:
Removed parameters and changed to stateless widget.